h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmLoss.java

```diff
@@ -106,25 +106,49 @@
 
 
   //--------------------------------------------------------------------------------------------------------------------
-  // Loss functions for binary features
+  // Loss functions for binary features read in as categorical or numeric
   //--------------------------------------------------------------------------------------------------------------------
 
   Logistic {
-    @Override public boolean isForNumeric() { return false; }
+    @Override public boolean isForNumeric() { return true; }
```

Comment:
Logistic loss cannot be applied to numeric columns! It's only defined for a = 0/1